Tests/POSIXTests/ReaddirTests.swift

```diff
@@ -38,7 +38,7 @@ class ReaddirTests: XCTestCase {
         
         do {
             var s = dirent()
-            let n = sizeof(s.d_name.dynamicType)
+            let n = MemoryLayout.of(s.d_name).size
```

Comment:
@gribozavr the type of `s` is platform dependent and something like `(Int8, Int8, Int8, ... * 256)`
Is there any way without using `._ofInstance(_:)` here?
```
#if os(Linux)
MemoryLayout<(Int8, Int8, Int8, Int8... * 245)>.size
#elif of(macOS)
MemoryLayout<(Int8, Int8, Int8, Int8, Int8... * 1024)>.size
...
#end
```
is not reasonable to me :(